src/tests/e2e/test-scenarios.ts

```diff
@@ -252,19 +252,8 @@ const testScenarios: TestScenarios = {
         { symbol: 'WBTC', exchangeRate: 0.1 },
       ],
     },
-    ARB2x: {
```

Comment:
Why did we remove this ? Is ARB2x no longer supported ? Unless that is the case I would  try to fix it instead of just removing the token from the tests entirely